### ISSUE TITLE
documentsDb does not correctly default to app server settings when left empty

### ISSUE THREAD

**Issue by bshalke** [marklogic/mlxprs#109]
The documentsDb setting should default to whatever is configured on the app server if left blank.  However, this error occurs instead:

```
{
  "message": "eval XQuery on server: cannot process response with 500 status",
  "statusCode": 500,
  "body": {
    "errorResponse": {
      "statusCode": 500,
      "status": "Internal Server Error",
      "messageCode": "INTERNAL ERROR",
      "message": "XDMP-RWEMPTY: The value of expression '$0' is required to be non-empty in rule: set-database . See the MarkLogic server error log for further detail."
    }
  }
}
```

**Comment by rjrudin** [comment]
@bshalke  I'll take a shot at this as a brief learning experience on how the plugin works. 

@anu3990  I may ping you about Node Client stuff since I haven't used that yet!

**Comment by mitchshepherd** [comment]
Addressed in 3.6.0 release
